fsm/leave_add_cds.go

```diff
@@ -17,7 +17,7 @@ var FsmLeaveAddCDS = music.FSMTransition{
 
 	PreCondition:  LeaveAddCDSPreCondition,
 	Action:        LeaveAddCDSAction,
-	PostCondition: func(z *music.Zone) bool { return true },
+	PostCondition: LeaveCDSVerify,
```

Comment:
If the pre-condition is called LeaveAddCDS... and the action is called LeaveAddCDS..., why isn't the post-condition called LeaveAddCDS...?